data/README.md

```diff
@@ -0,0 +1,11 @@
+# Data
+
+## Ordered protein structures
+
+The <code> rcsb_pdb_sequence*.csv</code> files are drawn from the RCSB Protein Data Bank (https://www.rcsb.org/). Two files were required because they were too large to be downloaded from the server as a single csv file. These structures contain exactly one protein chain and were solved using either x-ray diffraction, electron microscopy, or electron crystallography. Accessed November 9, 2021.
```

Comment:
You can also embed the link here and below .

Reply:
Done!